Examples/UICatalog/Scenarios/AnsiRequestsScenario.cs

```diff
@@ -112,7 +112,7 @@ private View BuildSingleTab ()
 
                                                       break;
                                                   case "CSI_ReportTerminalSizeInChars":
-                                                      selAnsiEscapeSequenceRequest = EscSeqUtils.CSI_ReportTerminalSizeInChars;
+                                                      selAnsiEscapeSequenceRequest = EscSeqUtils.CSI_ReportWindowSizeInChars;
```

Comment:
Should the case keyword allude the new name?
`case "CSI_ReportWindowSizeInChars":`